api/v1/testkube.yaml

```diff
@@ -8716,6 +8719,8 @@ components:
             - false
         tags:
           $ref: "#/components/schemas/TestWorkflowTagValue"
+        target:
+          $ref: "#/components/schemas/ExecutionTarget"
```

Comment:
There is already `runnerTarget` and `runnerOriginalTarget`.

Reply:
wdym? As for the component ref? These use the same references:

```
        runnerTarget:
          $ref: "#/components/schemas/ExecutionTarget"
        runnerOriginalTarget:
          $ref: "#/components/schemas/ExecutionTarget"
```